tests/selenium/pages/perfherder.py

```diff
@@ -6,15 +6,16 @@
                    Modal)
 
 
+# TODO fix tests to work with react
```

Comment:
I presume this means it'll come in a follow-up PR.  :)

Reply:
Yes, that'll be in the next pr. I'm also Ok with holding off on merging until the final part is converted and tests added (or fixed).